<ui version="4.0" >
 <class>lacaixa</class>
 <widget class="QDialog" name="lacaixa" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>lacaixa</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="keme.qrc" >
    <normaloff>:/lacaixa.png</normaloff>:/lacaixa.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <layout class="QFormLayout" name="formLayout" >
     <property name="fieldGrowthPolicy" >
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Cuenta encontrada:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="lineFound" >
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_9" >
       <property name="text" >
        <string>Sensibilidad:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <layout class="QHBoxLayout" name="horizontalLayout_4" >
       <item>
        <widget class="QSlider" name="sensibilidad" >
         <property name="minimum" >
          <number>1</number>
         </property>
         <property name="maximum" >
          <number>10</number>
         </property>
         <property name="pageStep" >
          <number>2</number>
         </property>
         <property name="value" >
          <number>6</number>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition" >
          <enum>QSlider::NoTicks</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10" >
         <property name="text" >
          <string>6</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_8" >
       <property name="text" >
        <string>Patr√≥n:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLineEdit" name="linePatron" >
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Cuenta en plan:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <widget class="QLineEdit" name="lineCuenta" />
       </item>
       <item>
        <widget class="QPushButton" name="buscapushButton" >
         <property name="minimumSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="keme.qrc" >
           <normaloff>:/images/librok-55.png</normaloff>:/images/librok-55.png</iconset>
         </property>
         <property name="shortcut" >
          <string/>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Fecha:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" >
      <widget class="QDateEdit" name="dateEdit" >
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Remitente:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" >
      <widget class="QLineEdit" name="lineRemitente" >
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0" >
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>Importe:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" >
      <widget class="QLineEdit" name="lineImporte" >
       <property name="text" >
        <string/>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="0" >
      <widget class="QLabel" name="label_6" >
       <property name="text" >
        <string>Tipo pase:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1" >
      <layout class="QHBoxLayout" name="horizontalLayout_2" >
       <item>
        <widget class="QCheckBox" name="checkGastos" >
         <property name="text" >
          <string>Gastos</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkIngresos" >
         <property name="text" >
          <string>Ingresos</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="8" column="0" >
      <widget class="QLabel" name="label_7" >
       <property name="text" >
        <string>Progreso lectura:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1" >
      <widget class="QProgressBar" name="progressBar" >
       <property name="value" >
        <number>0</number>
       </property>
       <property name="format" >
        <string>%p%</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <widget class="QPushButton" name="incorporarButton" >
       <property name="toolTip" >
        <string>Incorporar</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="keme.qrc" >
         <normaloff>:/images/librok-4.png</normaloff>:/images/librok-4.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ignorarButton" >
       <property name="toolTip" >
        <string>Ignorar siempre</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="keme.qrc" >
         <normaloff>:/images/librok-17.png</normaloff>:/images/librok-17.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="siguienteButton" >
       <property name="toolTip" >
        <string>Procesar</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="keme.qrc" >
         <normaloff>:/graf/1rightarrow.png</normaloff>:/graf/1rightarrow.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="toolTip" >
        <string>Salir</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="keme.qrc" >
         <normaloff>:/images/librok-7.png</normaloff>:/images/librok-7.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="status" >
     <property name="font" >
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="keme.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>lacaixa</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>443</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel" >
     <x>39</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sensibilidad</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_10</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>227</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel" >
     <x>436</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
